webview/src/experiments/components/table/header/TableHeader.tsx

```diff
@@ -27,7 +27,7 @@ const possibleOrders = {
 } as const
```

Comment:
The experiment column placeholder is showing a "Hide column" option: 

<img width="416" alt="image" src="https://user-images.githubusercontent.com/43496356/197220868-d9e489f9-5b59-400d-b9ec-016d96ff950b.png">

Reply:
I addressed this specific issue in #2643 , if that's ok